libcxx/test/lit.local.cfg

```diff
@@ -67,3 +67,17 @@ if (
         "%{link_flags}",
         os.path.join(build, "libc++std.a"),
     )
+
+    config.substitutions = appendToSubstitution(
+        config.substitutions,
+        "%{compile_flags}",
+        "-fprebuilt-module-path="
+        + os.path.join(
+            config.test_exec_root, "__config_module__/CMakeFiles/std.compat.dir"
+        ),
+    )
+    config.substitutions = appendToSubstitution(
+        config.substitutions,
+        "%{link_flags}",
+        os.path.join(build, "libc++std.compat.a"),
```

Comment:
For a simple follow-up: I think we should really rename those `.a` libraries. I know they're just implementation details, but the names are too close to `libstdc++` which could create confusion. I would suggest instead: `libc++.std.a` and `libc++.std.compat.a`? Or maybe something else but it's worth thinking about a name that is less easily confused with `libstdc++.a`.

Reply:
#76083 removes this code and its replacement #76246 only creates `pcm` files and not libraries. So that would resolve this issue.